src/main/java/edu/iu/terracotta/controller/app/TreatmentController.java

```diff
@@ -140,13 +137,12 @@ public ResponseEntity<Void> updateTreatment(@PathVariable("experiment_id") Long
         apijwtService.experimentAllowed(securedInfo, experimentId);
         apijwtService.treatmentAllowed(securedInfo, experimentId, conditionId, treatmentId);
 
-        if(apijwtService.isInstructorOrHigher(securedInfo)) {
-            return new ResponseEntity("Currently there is no functionality at this endpoint.", HttpStatus.OK);
-        }else {
+        if(!apijwtService.isInstructorOrHigher(securedInfo)) {
             return new ResponseEntity(TextConstants.NOT_ENOUGH_PERMISSIONS, HttpStatus.UNAUTHORIZED);
         }
-    }
 
+        return new ResponseEntity("Currently there is no functionality at this endpoint.", HttpStatus.OK);
```

Comment:
Should this call the TreatmentService.putTreatment method here?

Reply:
LOL! I'll blame the lack of coffee on this one! :)